Javascript/homework/responses/2.3-clock-design.md

```diff
@@ -4,5 +4,21 @@
 
 #### Task 3: Creating your clock
 
-Also define a clock class - this is where we will design a digital clock to print the time.
+After you're satisfied with the body of your page, define a block that styles elements in the `clock` class.
+
+![Screen Shot 2021-01-15 at 5.49.29 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-15 at 5.49.29 PM.png)
+
+First, set a width and a height for the box that's going to hold your time.  Add a background and a solid border using colors different from the screen's background so you can see what the box will look like.
+
+Once again, you'll want to set your `display` to `flex` and center your text using `align-items` and `justify-content` .  You need to do this again because these attributes in the body only centered the box on your page; repeating these lines in the clock section will center your text within the box.
+
+You may also want to adjust the margins of your clock so that theres some space between the box and the form - I simply adjusted the left margin of my clock.
+
+You can add as many bells and whistles to this clock that you please.  I increased the font-size here and added a box-shadow to add some depth to my clock.
+
+![Screen Shot 2021-01-15 at 6.17.26 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-15 at 6.17.26 PM.png)
+
+The first number in each line represents the x-offset which is the horizontal distance from which you want the object to appear.  The second number is the y-offset - the vertical distance from which the box will appear.  The third number is the blur radius - a larger blur radius will create a bigger shadow and vice versa. 
+
+Lastly is the color.  I used the `rgba()` tool to define my color.  The first 3 numbers represent the amount of red, green, and blue in the color, respectively. The fourth number is the opacity on a scale from 0 to 1, with 0 being complete transparency and 1 being fully opaque.
```

Comment:
What is the rgba tool and how do we use this?

Reply:
"The first 3 numbers represent the amount of red, green, and blue in the color, respectively." This explains it but not fully or how to open it.